webview/src/experiments/components/table/styles.module.scss

```diff
@@ -450,19 +454,57 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
         align-items: center;
         position: relative;
         left: -$cell-padding;
+        height: 100%;
 
         .starSwitch {
           display: inline-flex;
           align-items: center;
+          justify-content: center;
           width: 14px;
-          margin-left: 8px;
           opacity: 0.4;
           cursor: pointer;
 
           &[aria-checked='true'] {
             opacity: 1;
           }
         }
+
+        .indicatorIcon {
+          display: inline-flex;
+          align-items: center;
+          justify-content: center;
+          width: 1.3rem;
+          height: 100%;
+          padding: 0;
+          margin: 0;
+        }
+
+        .indicatorCount {
+          z-index: 2;
+          &[title='0'] {
+            display: none;
+          }
+        }
+
+        .bullet {
```

Comment:
I need to remove this, they are not used anymore

Reply:
Please remove this before merging